cmd/collector/app/collector_test.go

```diff
@@ -183,11 +179,11 @@ func TestCollector_PublishOpts(t *testing.T) {
 	require.NoError(t, c.Start(collectorOpts))
 	defer c.Close()
 
-	forkFactory.AssertGaugeMetrics(t, metricstest.ExpectedMetric{
+	metricsFactory.AssertGaugeMetrics(t, metricstest.ExpectedMetric{
 		Name:  "internal.collector.num-workers",
 		Value: 24,
 	})
-	forkFactory.AssertGaugeMetrics(t, metricstest.ExpectedMetric{
+	metricsFactory.AssertGaugeMetrics(t, metricstest.ExpectedMetric{
 		Name:  "internal.collector.queue-size",
```

Comment:
Where in the code are these emitted? If this test is passing we may have missed a place where these were emitted via "internal" namespace instead of writing directly to expvar

Reply:
yes, there is a metrics emitted from an internal ns